src/Simplex/Chat.hs

```diff
@@ -442,6 +438,18 @@ processChatCommand = \case
     checkSndFile f = do
       unlessM (doesFileExist f) . throwChatError $ CEFileNotFound f
       (,) <$> getFileSize f <*> asks (fileChunkSize . config)
+    updateProfile :: Profile -> Profile -> User -> m ChatResponse
+    updateProfile p p'@Profile {displayName} user = do
```

Comment:
There is probably no need to pass the old profile separately as it's inside User already